### ISSUE TITLE
vue3 error

### ISSUE THREAD

**Issue by wwhtml** [visualfanatic/vue-svg-loader#189]
error  in ./src/assets/svg/collapsed.svg

Syntax Error: Error:

Vue packages version mismatch:

- vue@3.2.32 (C:\Users\Wang\Desktop\新建文件夹 (2)\app\node_modules\vue\index.js)
- vue-template-compiler@2.6.14 (C:\Users\Wang\Desktop\新建文件夹 (2)\app\node_modules\vue-template-compiler\package.json)

This may cause things to work incorrectly. Make sure to use the same version for both.
If you are using vue-loader@>=10.0, simply update vue-template-compiler.
If you are using vue-loader@<10.0 or vueify, re-installing vue-loader/vueify should bump vue-template-compiler to the latest.

 @ ./node_modules/cache-loader/dist/cjs.js??ref--13-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader-v16/dist??ref--1-1!./src/App.vue?vue&type=script&lang=js 1:0-52 5:16-26
 @ ./src/App.vue?vue&type=script&lang=js
 @ ./src/App.vue
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://192.168.5.32:8080&sockPath=/sockjs-node (webpack)/hot/dev-server.js ./src/main.js

**Comment by mokone91** [comment]
Hi! vue-template-compiler is related to vue2, seems you need to use @vue/compiler-sfc

**Comment by weijian-zhu** [comment]
@vue/compiler-sfc does not work either  @mokone91

**Comment by linghengqian** [comment]
You need to provide a reproducible example.  If not, this issue should be closed and marked as invalid.

**Comment by CryptozombiesCorp** [comment]
Yeah this project no longer works with webpack 5. best to move to something else

**Comment by seyfer** [comment]
this is a good alternative https://github.com/shrpne/vue-inline-svg
